Fleece/MSVC_Compat.hh

```diff
@@ -38,6 +38,10 @@
     // WARNING: sizeof() will not work on this array since it's actually declared as a pointer.
     #define StackArray(NAME, TYPE, SIZE)    TYPE* NAME = (TYPE*)_malloca(sizeof(TYPE)*(SIZE))
 
+    #define strlcat strncat
```

Comment:
All the basic `<string.h>` functions are for single-byte encodings only, including UTF-8. There's a whole separate API using a `wchar_t` type for multi-byte characters (which I've never used.)